packages/oqamldebug/oqamldebug.0.9.5/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1"
+maintainer: "sebastien.fricker@gmail.com"
+build: [
+  [ "qmake" "CONFIG-=app_bundle" "CONFIG+=release" "PREFIX_BIN=%{bin}%/" "CONFIG-=debug" "oqamldebug.pro" ]
+  ["%{make}%"]
+  ["%{make}%" "install" ]
+]
+remove: [
+  [ "qmake" "CONFIG-=app_bundle" "CONFIG+=release" "PREFIX_BIN=%{bin}%/" "CONFIG-=debug" "oqamldebug.pro" ]
+  [ "make" "uninstall" ]
```

Comment:
On 26/07/2015 20:29, David Sheets wrote:
>
> In packages/oqamldebug/oqamldebug.0.9.5/opam
> <https://github.com/ocaml/opam-repository/pull/4366#discussion_r35495851>:
>
> > @@ -0,0 +1,12 @@
> > +opam-version: "1"
> > +maintainer: "sebastien.fricker@gmail.com"
> > +build: [
> > +  [ "qmake" "CONFIG-=app_bundle" "CONFIG+=release" "PREFIX_BIN=%{bin}%/" "CONFIG-=debug" "oqamldebug.pro" ]
> > +  ["%{make}%"]
> > +  ["%{make}%" "install" ]
> > +]
> > +remove: [
> > +  [ "qmake" "CONFIG-=app_bundle" "CONFIG+=release" "PREFIX_BIN=%{bin}%/" "CONFIG-=debug" "oqamldebug.pro" ]
> > +  [ "make" "uninstall" ]
>
> 2
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ocaml/opam-repository/pull/4366/files#r35495851>.
>

This is really not a good solution.
If %{make}% is used, the user has the possibility to override it with
its own version of make command line with the environment variable
OPAMVAR_make.
What you suggest will force to use the 'make' command line tool in all
cases.